DocumentedExamples/regional-mom6-forced-by-access-om2.ipynb

```diff
@@ -0,0 +1,701 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    om2_input = xr.open_mfdataset(
</pre><p>can we use the cookbook to load this data?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/338/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
Actually, better we use the intake catalog! @luweiyang could you make this change at this line instead of opening a hardcoded netCDF file via `xarray.open_mfdataset`?